aten/src/ATen/native/cpu/PointwiseOpsKernel.cpp

```diff
@@ -90,16 +91,50 @@ static void addcdiv_cpu_kernel(TensorIteratorBase& iter, const Scalar& value) {
   }
 }
 
+void l1_backward_cpu_kernel(TensorIterator& iter, const Scalar& norm) {
+  if (isComplexType(iter.dtype())) {
+    AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "l1_backward_cpu_complex", [&]() {
+      using Vec = Vectorized<scalar_t>;
+      const auto norm_val = norm.to<scalar_t>();
+      const auto norm_val_vec = Vec(norm_val);
+      cpu_kernel_vec(iter,
+        [norm_val](scalar_t input, scalar_t target, scalar_t grad_output) -> scalar_t {
+          return sgn_impl(input - target) * norm_val * grad_output;
+        },
+        [norm_val_vec](Vec input, Vec target, Vec grad_output) -> Vec {
+          return (input - target).sgn() * norm_val_vec * grad_output;
+        });
+    });
+  } else {
+    AT_DISPATCH_FLOATING_TYPES_AND2(kBFloat16, kHalf, iter.dtype(), "l1_backward_cpu_real", [&]() {
+      using Vec = Vectorized<scalar_t>;
+      const auto norm_val = norm.to<scalar_t>();
+      const auto norm_val_vec = Vec(norm_val);
+      const auto zero_vec = Vec(static_cast<scalar_t>(0));
+      cpu_kernel_vec(iter,
+        [norm_val](scalar_t input, scalar_t target, scalar_t grad_output) -> scalar_t {
+          const auto x = input - target;
+          return static_cast<scalar_t>(((0 < x) - (x < 0))) * norm_val * grad_output;
+        },
+        [norm_val_vec, zero_vec](Vec input, Vec target, Vec grad_output) -> Vec {
+          const auto x = input - target;
+          const auto y = norm_val_vec * grad_output;
+          return Vec::blendv(y.neg(), y, x > zero_vec);
```

Comment:
Forgot the zero case here. This should fix it.
```suggestion
          return Vec::blendv(Vec::blendv(y.neg(), y, x > zero_vec), zero_vec, x == zero_vec);
```